conduit/plugins/importers/algod/mock_algod_test.go

```diff
@@ -33,7 +33,11 @@ func NewAlgodHandler(responders ...algodCustomHandler) *AlgodHandler {
 
 // ServeHTTP implements the http.Handler interface for AlgodHandler
 func (handler *AlgodHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
-	for _, responder := range handler.responders {
+	for i, responder := range handler.responders {
+		_ = i
+		if responder == nil {
```

Comment:
was panicking previously on unimplemented responders

Reply:
Thanks for fixing this. I noticed it the last time I was working on this test but the complexity really wore me down and I decided to leave it. This fix might make the other logging changes unnecessary, since the panic output would no longer be a distraction.

Reply:
I'm removing an extra log, but keeping the "WE HAVE A PROBLEM" under a new condition.